django/contrib/auth/models.py

```diff
@@ -305,7 +305,7 @@ class AbstractUser(AbstractBaseUser, PermissionsMixin):
         },
     )
     first_name = models.CharField(_('first name'), max_length=30, blank=True)
-    last_name = models.CharField(_('last name'), max_length=150, blank=True)
+    last_name = models.CharField(_('last name'), max_length=150, blank=True, unique =True,)
     email = models.EmailField(_('email address'), blank=True)
```

Comment:
you might want to as unique=True to email field?

Reply:
That's my pull request. Emails are meant to be unique, is there a case scenario where they won't be?

Reply:
look, you added unique true to last name not in email

Reply:
oh, my bad. I've made the correction already